docs/errors.md

```diff
@@ -217,6 +217,53 @@ Selectors on the [NSObjectProtocol](https://developer.apple.com/reference/object
 
 Note: The list of reserved selectors will evolve with new versions of the tool.
 
+<h3><a name="EM1060"/>Invalid `RegisterAttribute` found on class `T`. Expected the first constructor parameter to be of type `string` but found type `U` instead.</h3>
+
+This is a **warning** that the class `T` was decorated with a `RegisterAttribute`, indicating the author wishes to rename the name of generated objective-c class.
```

Comment:
Minor: use proper casing `Objective-C` instead of all lowercase (there are also more cases below).